src/main/StreamTabsScreen.js

```diff
@@ -44,35 +48,38 @@ type Props = $ReadOnly<{|
 
 export default function StreamTabsScreen(props: Props) {
   return (
-    <Tab.Navigator
-      {...materialTopTabNavigatorConfig({
-        showLabel: true,
-        showIcon: false,
-      })}
-      swipeEnabled
-    >
-      <Tab.Screen
-        name="subscribed"
-        component={SubscriptionsCard}
-        options={{
-          tabBarLabel: ({ color }) => (
-            <Text style={[styles.tab, { color }]}>
-              <FormattedMessage id="Subscribed" defaultMessage="Subscribed" />
-            </Text>
-          ),
-        }}
-      />
-      <Tab.Screen
-        name="allStreams"
-        component={StreamListCard}
-        options={{
-          tabBarLabel: ({ color }) => (
-            <Text style={[styles.tab, { color }]}>
-              <FormattedMessage id="All streams" defaultMessage="All streams" />
-            </Text>
-          ),
-        }}
-      />
-    </Tab.Navigator>
+    <View style={styles.container}>
+      <Tab.Navigator
+        {...materialTopTabNavigatorConfig({
+          showLabel: true,
+          showIcon: false,
+        })}
+        swipeEnabled
+      >
+        <Tab.Screen
+          name="subscribed"
+          component={SubscriptionsCard}
+          options={{
+            tabBarLabel: ({ color }) => (
+              <Text style={[styles.tab, { color }]}>
+                <FormattedMessage id="Subscribed" defaultMessage="Subscribed" />
+              </Text>
+            ),
+          }}
+        />
+        <Tab.Screen
+          name="allStreams"
+          component={StreamListCard}
+          options={{
+            tabBarLabel: ({ color }) => (
+              <Text style={[styles.tab, { color }]}>
+                <FormattedMessage id="All streams" defaultMessage="All streams" />
+              </Text>
+            ),
+          }}
+        />
+      </Tab.Navigator>
+      <LoadingBanner />
```

Comment:
It looks like this will put the loading bar/banner at the bottom of this view, instead of the top. Is that intended? Can you post a gif of what that looks like?

Reply:
No it will be at top, as LoadingBar has `style={{position: 'absolute'}}`, and yes its intended, as it is the last component and has `position:absolute` it will be in most upper layer(It will have higher zIndex)

Reply:
Hmm, I see.

Would you post a screencap showing what this looks like on that screen?